tests/fips/openssl/openssl_fips_alglist.pm

```diff
@@ -60,10 +60,14 @@ sub run {
     # The openssl command is adjustment
     else {
         # Add 3DES and 3des support
-        validate_script_output
+        eval {
+            validate_script_output(
 "echo -n 'Invalid Cipher: '; openssl list -cipher-algorithms | sed -e '/AES/d' -e '/aes/d' -e '/DES3/d' -e '/des3/d' -e '/DES-EDE/d' -e '/3DES/d' -e '/3des/d' | wc -l",
-          sub { m/^Invalid Cipher: 0$/ };
-
+                sub { m/^Invalid Cipher: 0$/ }); };
+        if ($@) {
+            # It is not important function, just record it
+            # POO#111818
+            record_info('bsc#1161276', 'It is not important function about openssl list -cipher-algorithms, and marked this as WONTFIX'); }
```

Comment:
I would suggest you use record_softfail since we may need this in future releases :)